src/shared/table-options/CustomFilter.js

```diff
@@ -2,20 +2,47 @@ import React from 'react'
 import Icon from '../site-config/Icon'
 
 const CustomFilter = (props) => {
+  const {
+    setFilterModalStatus,
+    setFilteredRows,
+    setFilteredHeaders,
+    isFiltered,
+    setIsFiltered,
+    allRows,
+    allHeaders
+  } = props
+
   const handleClick = () => {
-    props.setFilterModalStatus(true)
+    setFilterModalStatus(true)
+  }
+
+  const handleResetFilters = () => {
+    setFilteredRows(allRows)
+    setFilteredHeaders(allHeaders)
+    setIsFiltered(false)
+  }
+
+  const renderClass = () => {
+    return isFiltered
```

Comment:
This could be written in a more DRY way so we don't have to write the whole list of classes twice. That could look like this:

```jsx
let filteredClass = isFiltered ? ' js-isActive' : ''
return 'btn btn--dark btn--icon filter__button' + filteredClass
```